lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -99,10 +100,28 @@ def write_playbook
 
       def start_ansible_runner
         command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        setup_verbosity(command) if verbose?
```

Comment:
I do not like the name `setup` too much xD WDYT about this:
```suggestion
        command << verbosity_option if verbose?
```

Reply:
@ezr-ondrej thank you so much for the overall feedback. With all your suggestions embedded, my commit looks more clean and tweaked. :green_heart: 

I tested this on a Satellite 6.7 server and it works :thumbsup: